### PR TITLE
Add gallery feature to pet images

### PR DESCRIPTION
**Pull request by shandyk18**

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -44,9 +43,23 @@ <h3>Favorite Projects I Have Worked On</h3>
...
+          <div class="slide">
+            <img src="images/dog1.jpg" alt="Luci" width="650" height="850">
+          </div>
+          <div class="slide">
+            <img src="images/dog2.jpg" alt="Luci with a flower" width="650" height="850">
+          </div>
+          <div class="slide">
+            <img src="images/dog3.jpg" alt="Luci as a puppy" width="650" height="850">
+          </div>
+
```
[the remark this answers is not in the document]
Comment (shandyk18) [reply portfolio/src/main/webapp/index.html:58]: For some reason the slide class isn't changing the images themselves. I created a new dog-image class.

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -12,12 +12,30 @@
...
+var slideIndex = 1;
+showSlides(slideIndex);
+
+// Next/previous controls
+function nextSlide(n) {
+  showSlides(slideIndex += n);
+}
+
+function showSlides(n) {
+  var slides = document.getElementsByClassName("slide");
```
[the remark this answers is not in the document]
Comment (shandyk18) [reply portfolio/src/main/webapp/script.js]: Got it!

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -12,12 +12,30 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+var slideIndex = 1;
```
[the remark this answers is not in the document]
Comment (shandyk18) [reply portfolio/src/main/webapp/script.js]: Got it!

File: portfolio/src/main/webapp/index.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (MorganAw) [reply portfolio/src/main/webapp/index.html:58]: you could maybe try doing
```
.slide > img {
  width: 650px;
  height: 850px;
}
```

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -12,12 +12,31 @@
...
+// Next/previous controls
+function nextSlide(n) {
+  showSlides(slideIndex += n);
+}
+
+function showSlides(n) {
+  let slides = document.getElementsByClassName("slide");
+  if (n > slides.length) {slideIndex = 1}
+  if (n < 1) {slideIndex = slides.length}
+  for (var i = 0; i < slides.length; i++) {
```
Comment (MorganAw) [inline portfolio/src/main/webapp/script.js:27]: nit: this can also be a `let` declaration

**Pull request merged**